generators/wizard/templates/wizard.ts

```diff
@@ -77,16 +80,32 @@ import * as ng from "@angular/core";
     outputs: [],
     templateUrl: './<%= wizard.name %>.html',
     styleUrls: ["./<%= wizard.name %>.css"],
-    assign: { i18n : i18n }
+    assign: { i18n: i18n }
 })
-export class <%= wizard.class %> extends<% if (wizard.isExtendingMes) { %> MesComponent <% } %><% if (!wizard.isExtendingMes) { %> CoreComponent <% } %>implements ng.OnChanges, TransactionWizardInterface {
+export class <%= wizard.class %> extends <% if (wizard.isExtendingMes) { %>MesComponent<% } %> <% if (!wizard.isExtendingMes) { %>CoreComponent <% } %>implements TransactionWizardInterface {
+
+    /**
+     * The wizard element
+     */
+    @ng.ViewChild(Wizard)
+    protected _nestedWizard: Wizard;
 
     //#region Private properties
 
     //#endregion
 
     //#region Public properties
 
+    /**
```

Comment:
It appears to me that there is no need to have 2 instances. The only reason for having both would be if we were assigning the instance on the constructor (from the page bag), and then re assigning it